### PR TITLE
chore(data): added queue visualizer

### PR DESCRIPTION
**Pull request by UG-SEP**
Hey @ming-tsai ,
Please review it 
Thanks

### CODE REVIEW

**Comment by ming-tsai** [comment]
Hi @UG-SEP,
Does the repo is on private?

**Comment by UG-SEP** [comment]
Sorry for that [Link](https://github.com/UG-SEP/Queue-Visualizer)

File: data/projects.json [hunk 1]
```diff
@@ -108,5 +108,15 @@
         "level": "basic",
         "site": "",
         "tags": "feature-maps-and-filter-visualizer, machine-learning, python"
+    },
+     {
+        "name": "queue_visualizer",
```
Comment (ming-tsai) [inline data/projects.json:113]:
```suggestion
        "name": "Queue-Visualizer",
```

Comment (ming-tsai) [reply data/projects.json:113]: Hi @UG-SEP,
Please change replace `_` to `-`

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**